ONSdigital/ras-collection-instrument
# adding ignore

# What and why?
This PR adds an ignore to the makefile.

# How to test?
Deploy and run tests.

Do pipenv install, pipenv graph, and check following dependencies:

flask-cors = “>=4.0.2”
requests = “>=2.32.0”

# Jira
[Card](https://jira.ons.gov.uk/browse/RAS-1315)
